### PR TITLE
Added missing dependencies

### PR DESCRIPTION
**Pull request by core23**
Added all used dependencies to the `composer.json` file.

Usage:
https://github.com/FriendsOfSymfony/FOSUserBundle/blob/master/FOSUserBundle.php#L14-L16

### CODE REVIEW

**Comment by XWB** [comment]
Tests fail.

**Comment by XWB** [comment]
Tests still fail.

**Comment by core23** [comment]
Depends on https://github.com/FriendsOfSymfony/FOSUserBundle/pull/2243

File: composer.json [hunk 1]
```diff
@@ -27,7 +27,12 @@
         "symfony/twig-bundle": "^2.3 || ^3.0"
     },
     "require-dev": {
+        "doctrine/couchdb": "^1.0.0@dev",
+        "doctrine/couchdb-odm": "^1.0.0@dev",
+        "doctrine/couchdb-odm-bundle": "^2.1.0@dev",
         "doctrine/doctrine-bundle": "^1.3",
+        "doctrine/orm": "^2.5.5",
+        "doctrine/mongodb-odm": "^1.1.1",
```
Comment (stof) [inline composer.json]: this makes it impossible for users to install PHPUnit locally if they don't have the mongo extension (and it does not exist on PHP 7), so this looks like a bad idea to me: it makes collaboration harder

Comment (stof) [inline composer.json]: btw, see Travis to see that this is also a bad idea

Comment (stloyd) [inline composer.json]: This should be added directly in `.travis.yml` file as `composer require --dev` to not break anything.

Comment (core23) [inline composer.json]: > this makes it impossible for users to install PHPUnit locally if they don't have the mongo extension (and it does not exist on PHP 7), so this looks like a bad idea to me: it makes collaboration harder

This is no change for the `user`, only for developers who want to contribute.

**Pull request closed without merging**